model/src/test/java/uk/gov/pay/commons/model/DateTimeFormattersTest.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.pay.commons.model;
+
+
+import org.junit.Test;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+public class DateTimeFormattersTest {
+
+    @Test
+    public void shouldFormatToMillisecondPrecision() {
+        ZonedDateTime aZonedDateTime = ZonedDateTime.of(1969, 7, 20, 20, 18, 4, 111111111, ZoneId.of("UTC"));
+        assertThat(aZonedDateTime.format(DateTimeFormatters.MILLISECOND_ISO_INSTANT_FORMATTER), is("1969-07-20T20:18:04.111Z"));
```

Comment:
Don't we need a method to format date to this pattern ?

Reply:
As discussed its intended use is with `.format()` method on `ZonedDateTime` so a static method to take a `ZonedDateTime` and return a formatted String isn't necessary (yet).